packages/@aws-cdk/aws-s3-deployment/lambda/src/requirements.txt

```diff
@@ -1,2 +1,2 @@
-awscli==1.16.34
-boto3==1.9.177
```

Comment:
Might want those to match what's actually on the Lambda runtimes: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html -- or are those requirements superseding that?

Reply:
Those supersede intentionally